.github/workflows/aws_nightly_cleanup.yml

```diff
@@ -77,18 +94,6 @@ jobs:
                   --exclude-resource-type s3 \
                   --exclude-resource-type cloudtrail || true
 
-            # Following will delete regional resources that cloud-nuke does not support
-            - name: Delete additional regional AWS resources
-              timeout-minutes: 15
-              run: .github/workflows/scripts/aws_regional_cleanup.sh "${{ env.AWS_REGION }}"
-
-            # Following will delete global resources that cloud-nuke does not support
-            - name: Delete additional global AWS resources
-              # Only run in a single matrix run
-              if: ${{ env.AWS_REGION == 'eu-west-2' }}
-              timeout-minutes: 15
-              run: .github/workflows/scripts/aws_global_cleanup.sh
-
```

Comment:
my suggestion would be to keep the order of execution.

Cloud-nuke could delete a EKS cluster, which would free up IAM roles / policies, which would otherwise be still bound.

Reply:
my assumption is that if an EKS / ROSA cluster exists it would fail if that wasn't cleaned up prior as those roles are afaik bound to the cluster.
If you say it also fails in the other order, then we would probably not win anything either way 😅.

I'm fine keeping it as is for now and reevaluate when it fails.